translation works

Danfro opened the pull request:
This PR will
- add missing string (Hide/Show %1 expired pass)
- fix )) typos where only one bracket is needed
- update German translation

patrickjane commented:
@Danfro Hey, I have merged all your PRs, but unfortunately, I no longer have a phone running UT, so I cannot test the app. The desktop mode is not suited for testing layouts & stuff. So would you be so kind to test the changes? If everything looks okay on your side, I will push the update to the store.

Download the app here: https://github.com/patrickjane/ut-passes/releases/tag/1.3.2-RC

Danfro commented:
Hi @patrickjane I am running the app for a while now including my changes, so fine on my end - except of the issues reported.

Although I just tried to build from main and got an error:
```
CMake Error in CMakeLists.txt:
  Imported target "ZXing::Core" includes non-existent path

    "/./include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in CMakeLists.txt:
  Imported target "ZXing::Core" includes non-existent path

    "/./include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.
```
Can that be due to the one PR from Martin you merged?

EDIT: I did update the submodules and build the libs again.

Danfro commented:
I did drop a request for testing also in our QA group. Lets see if someone has time to test.

patrickjane commented:
Well, I just built it with all the changed merged, abd I did not have that error. I might have a different submodule checkout tho 🤔
I think I might even have had manual changes on the submodules, which must be applied, but not sure. I set this up long ago. Will check tomorrow or so.

Danfro commented:
I just saw your [patch](https://github.com/patrickjane/ut-passes/blob/main/zxing_patch.patch). Maybe you changed something there that wasn't pushed to the repository yet?

patrickjane commented:
No, I don't have any local changes in the patch. I am on commit e3556b5 of zxing. Maybe you're on a different one? Did one of your changes update zxing submodule, and I did not yet update?

Danfro commented:
I am on v1.0.8 of zxing-cpp (85f9eeb4ce8f91d8fb732e1ab00243a6527bb414), so that might be difference we are chasing. But this version is what I get when clearing the libs folder and doing `git submodule update --init`. Not sure why it doesn't pick a more recent version though. Not sure where the version is tagged or why that is different on your end.
